src/TestExtractTestDuration/TestExtractTestDuration/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text.RegularExpressions;
+
+var r = new Regex("(2022.*?\\+0000).*?Uno.UI.Samples.Tests.UnitTestsControl\\[0\\]");
+
+var lines = File.ReadAllLines(@"C:\Users\jerome.laban\Downloads\DeviceLog-4-20220318041159.txt");
```

Comment:
Was this new TestExtractTestDuration sln supposed to be added?

Reply:
nope :)